lib/modules/manager/circleci/range.spec.ts

```diff
@@ -0,0 +1,14 @@
+import type { RangeConfig } from '../types';
+import { getRangeStrategy } from './range';
```

Comment:
```suggestion
import { getRangeStrategy } from '.';
```

better code coverage 😉